testing/test_aux.out

```diff
@@ -107,4 +116,5 @@ port-02 module_config modules
                 ')
 Redacted docs diff
 No report diff
-Done with tests
\ No newline at end of file
+ M docs/device_report.md
```

Comment:
@ewRedstone (This is why Travis is failing)

Reply:
Yeah its weird, extra 15 minutes this is not the most productive use of time